.github/dockerfiles/e2e/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+services:
+  sproxyd:
+    profiles: ['s3c']
+    build:
+      context: ../sproxyd
+    ports:
+      - 8181
```

Comment:
is it used "externally" ? if sproxyd is only used from the other containers, this is not needed